hecuba_core/src/ArrayDataStore.h

```diff
@@ -18,24 +18,26 @@ class ArrayDataStore {
 
     ~ArrayDataStore();
 
-    void store(const uint64_t *storage_id, ArrayMetadata* metadata, void *data) const;
-
+    void store(const uint64_t *storage_id, ArrayMetadata *metadata, void *data) const;
+    void store_numpy_to_cas(const uint64_t *storage_id, ArrayMetadata *metadata, void *data, std::list<std::vector<uint32_t> > coord) const;
```

Comment:
Please put a newline :)

coord is a list of vectors, i think you can pass that as reference( &) and should be more efficient. you can add the const to the list to be sure it is not modified inside the program if you want.

Besides, why do you pass the coord list if it is not used inside the method? Maybe you can have two methods. E.g:
store_array()
store_subarray()